### ISSUE TITLE
Error "Failed to list *v1.Service: services "" is forbidden: User "system:node:127.0.0.1" cannot list services at the cluster scope. "

### ISSUE THREAD

**Issue by mtanino** [kubernetes/kubernetes#49822]
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind bug

**What happened**:
After following commit was merged, 'kubectl get node' can't find local node due to following error.
I'm using hack/local-up-cluster.sh for installation tool.

```
commit 3d3d3922c21ce5e81f7c94ff7ef0bbbc9b4e0b40
Date:   Fri Jul 28 05:08:10 2017 -0700
```
```
# k get node
No resources found.
```
```
I0728 19:22:15.081807   17992 reflector.go:234] Listing and watching *v1.Service from k8s.io/kubernetes/pkg/kubelet/kubelet.go:389 
I0728 19:22:15.082839   17992 reflector.go:234] Listing and watching *v1.Pod from k8s.io/kubernetes/pkg/kubelet/config/apiserver.go:47 
E0728 19:22:15.083905   17992 reflector.go:199] k8s.io/kubernetes/pkg/kubelet/config/apiserver.go:47: Failed to list *v1.Pod: pods "" is forbidden: User "system:node:127.0.0.1" cannot list pods at the cluster scope. 
E0728 19:22:15.084267   17992 reflector.go:199] k8s.io/kubernetes/pkg/kubelet/kubelet.go:389: Failed to list *v1.Service: services "" is forbidden: User "system:node:127.0.0.1" cannot list services at the cluster scope. 
I0728 19:22:15.090334   17992 reflector.go:234] Listing and watching *v1.Node from k8s.io/kubernetes/pkg/kubelet/kubelet.go:398 
E0728 19:22:15.091403   17992 reflector.go:199] k8s.io/kubernetes/pkg/kubelet/kubelet.go:398: Failed to list *v1.Node: nodes "" is forbidden: User "system:node:127.0.0.1" cannot list nodes at the cluster scope. 
I0728 19:22:16.084029   17992 reflector.go:234] Listing and watching *v1.Pod from k8s.io/kubernetes/pkg/kubelet/config/apiserver.go:47 
E0728 19:22:16.084840   17992 reflector.go:199] k8s.io/kubernetes/pkg/kubelet/config/apiserver.go:47: Failed to list *v1.Pod: pods "" is forbidden: User "system:node:127.0.0.1" cannot list pods at the cluster scope. 
I0728 19:22:16.085081   17992 reflector.go:234] Listing and watching *v1.Service from k8s.io/kubernetes/pkg/kubelet/kubelet.go:389 
E0728 19:22:16.085652   17992 reflector.go:199] k8s.io/kubernetes/pkg/kubelet/kubelet.go:389: Failed to list *v1.Service: services "" is forbidden: User "system:node:127.0.0.1" cannot list services at the cluster scope. 
```

**What you expected to happen**:

Node can be found.

**How to reproduce it (as minimally and precisely as possible)**:
1. hack/local-up-cluster.sh
1. kubectl.sh get node

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): latest master
- Install tools: local-up-cluster.sh

**Comment by mtanino** [comment]
/sig node
@liggitt

**Comment by sbezverk** [comment]
I am seeing the same issue.

**Comment by liggitt** [comment]
fixed in #49812
